### PR TITLE
Fix tests and Migration update

### PR DESCRIPTION
**Pull request by ruromero**
Signed-off-by: Ruben Romero Montes <rromerom@redhat.com>

* Unit tests were failing due to updated hibernate library
* Fixed problem while updating Migration instances

### CODE REVIEW

**Comment by kiereleaseuser3** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

**Comment by ruromero** [comment]
CC @MarianMacik

**Comment by MarianMacik** [comment]
ok to test

**Comment by MarianMacik** [comment]
@mswiderski any comments?

File: process-migration-service/src/test/java/org/kie/processmigration/persistence/TestEntityManager.java [hunk 1]
```diff
@@ -0,0 +1,289 @@
+package org.kie.processmigration.persistence;
```
Comment (mswiderski) [inline process-migration-service/src/test/java/org/kie/processmigration/persistence/TestEntityManager.java]: missing license header

**Comment by MarianMacik** [comment]
@mswiderski There is a header template configured in kie-parent: `.*` which fits for everything. Moreover, in kie-parent we have: `<checkstyle.failOnViolation>false</checkstyle.failOnViolation>`. In jBPM we have quite largely overridden configuration so it just checks templates and nothing else. If I turn failOnViolation on, there are ~130 violations just in process-instance-migration module. Finally, I have noticed that process-instance-migration is not built by default, see:
https://github.com/kiegroup/droolsjbpm-integration/blob/947db7e178ed6d6330adb264dd7a0d9093c77a9e/pom.xml#L52-L81

So that's why :) I would turn on failOnViolation and fix the issues in whole repo, but certainly not this week. Wdyt?

**Comment by mswiderski** [comment]
thanks @MarianMacik, that explains why I can't find it in any release in maven repo, +1 at enabling that whenever you have the time.

File: process-migration-service/src/test/java/org/kie/processmigration/persistence/TestEntityManager.java (hunk 1 above)
Comment (ruromero) [reply process-migration-service/src/test/java/org/kie/processmigration/persistence/TestEntityManager.java]: Done, thanks

**Pull request merged**